docs-md/basics/cli-commands.md

```diff
@@ -0,0 +1,218 @@
+---
+title: CLI Commands 
+description: Capacitor CLI command reference list
+url: /docs/basics/cli-commands
+contributors:
+  - dotnetkow
+---
+
+# Capacitor CLI Reference
+
+<p class="intro">The Capacitor command-line interface (CLI) tool is used to develop Capacitor apps.</p>
+
+## Installation
+
+Capacitor does not use a global CLI. Instead, CLI is installed locally into each project as an npm script. This makes it easier to manage versions of Capacitor across many different apps.
+
+```bash
+cd my-app
+npm install @capacitor/core @capacitor/cli
+```
```

Comment:
Is this necessary or can we link to the installation page?

Reply:
true... removed the heading but mention it in the beginning sentence.